Source/BugsnagUser.m

```diff
@@ -14,7 +14,7 @@ @implementation BugsnagUser
 - (instancetype)initWithDictionary:(NSDictionary *)dict {
     if (self = [super init]) {
         _userId = dict[@"id"];
-        _emailAddress = dict[@"emailAddress"];
+        _emailAddress = dict[@"email"];
```

Comment:
are the changes in this file related to this PR?

Reply:
This addresses an issue in the session request serialisation which was found when adding mazerunner scenarios. Currently the session request sends `user.emailAddress` rather than `user.email`, which is corrected in e12ff70.

I can split this off into a separate PR if you think it makes more sense.

Reply:
that makes sense, I think it would be good to have it in a separate PR to track it more easily